pynxtools_xps/scienta/scienta_txt.py

```diff
@@ -32,16 +34,48 @@
     construct_entry_name,
     construct_data_key,
     construct_detector_data_key,
+    convert_pascal_to_snake,
 )
 
+from pynxtools_xps.scienta.scienta_txt_data_model import ScientaHeader, ScientaRegion
+
+KEY_MAP: Dict[str, str] = {
+    "number_of_regions": "no_of_regions",
+    "version": "software_version",
+    "dimension_name": "energy_units",
+    "dimension_size": "energy_size",
+    "dimension_scale": "energy_axis",
+    "number_of_sweeps": "no_of_scans",
+    "energy_unit": "energy_scale_2",
+    "low_energy": "start_energy",
+    "high_energy": "stop_energy",
+    "energy_step": "step_size",
+    "step_time": "dwell_time",
+    "detector_first_x-_channel": "detector_first_x_channel",
+    "detector_last_x-_channel": "detector_last_x_channel",
+    "detector_first_y-_channel": "detector_first_y_channel",
+    "detector_last_y-_channel": "detector_last_y_channel",
+    "file": "data_file",
+    "sequence": "sequence_file",
+    "spectrum_name": "spectrum_type",
+    "instrument": "instrument_name",
+    "location": "vendor",
+    "user": "user_name",
+    "sample": "sample_name",
+    "comments": "spectrum_comment",
+    "date": "start_date",
+    "time": "start_time",
+    "transmission": "fixed analyzer transmission",
+}
```

Comment:
Can/should this not be part of the config file?

Reply:
I mainly do this to enforce similar names already on the parser level, so it is more clean if you look at the config what the data is that is being collected. It is not really necessary, but it makes it easier for me to maintain the default config files when the nexus definitions change.

Reply:
I understand. That makes sense